launch/tier4_control_launch/launch/control.launch.py

```diff
@@ -429,17 +468,24 @@ def add_launch_arg(name: str, default_value=None, description=None):
 
     # component
     add_launch_arg("use_intra_process", "false", "use ROS 2 component container communication")
-    add_launch_arg("use_multithread", "true", "use multithread")
+    add_launch_arg("use_multithread", "false", "use multithread")
+    add_launch_arg(
+        "trajectory_follower_mode",
+        "trajectory_follower_node",
+        "Options for which trajectory_follower to use. Options: `trajectory_follower_node`, `smart_mpc_trajectory_follower`",
+    )
     set_container_executable = SetLaunchConfiguration(
         "container_executable",
         "component_container",
         condition=UnlessCondition(LaunchConfiguration("use_multithread")),
     )
-    set_container_mt_executable = SetLaunchConfiguration(
-        "container_executable",
-        "component_container_mt",
-        condition=IfCondition(LaunchConfiguration("use_multithread")),
-    )
+
+    if True:
+        set_container_mt_executable = SetLaunchConfiguration(
+            "container_executable",
+            "component_container_mt",
+            condition=IfCondition(LaunchConfiguration("use_multithread")),
+        )
```

Comment:
if condition is not needed?

```suggestion
    set_container_mt_executable = SetLaunchConfiguration(
        "container_executable",
        "component_container_mt",
        condition=IfCondition(LaunchConfiguration("use_multithread")),
    )
```
?